deploy/terraform-aws/node.sh

```diff
@@ -68,7 +73,9 @@ controllerManager:
     address: 0.0.0.0
 kubernetesVersion: "$k8s_version"
 EOF
+kubeadm config migrate --old-config /tmp/old-kubeadm-config.yaml --new-config /tmp/kubeadm-config.yaml
 kubeadm init --config=/tmp/kubeadm-config.yaml
+stat /etc/kubernetes/admin.conf
```

Comment:
I think you can remove this when `set -e` is set at the top of the script. It kubeadm can't create the configuration it will certainly exit with a non-zero error code.